test/spec/api/create-vis.spec.js

```diff
@@ -281,7 +281,7 @@ describe('src/api/create-vis', function () {
 
   var mapInstantiationRequestDone = function () {
     return _.any($.ajax.calls.allArgs(), function (args) {
-      var expectedURLRegexp = /(http|https):\/\/cdb.localhost.lan:8181\/api\/v1\/map\/named\/tpl_6a31d394_7c8e_11e5_8e42_080027880ca6\?stat_tag=6a31d394-7c8e-11e5-8e42-080027880ca6/;
+      var expectedURLRegexp = /(http|https):\/\/cdb.localhost.lan:8181\/api\/v1\/map\/named\/tpl_6a31d394_7c8e_11e5_8e42_080027880ca6\?/;
```

Comment:
I changed the order of params in this PR and it started to failed so I decided to "relax" the expectation a bit. Whenever I use this helper in the tests, all I want to ensure is that a request to the Maps API was performed (there are other tests to ensure URLs are generated correctly [here](https://github.com/CartoDB/cartodb.js/blob/4eb7182dc406559215b64b1295dc6d6143d153f9/test/spec/windshaft/client.spec.js)).